pygfx/objects/_more.py

```diff
@@ -28,51 +80,125 @@ def __init__(self, geometry=None, material=None, **kwargs):
 
 
 class Line(WorldObject):
-    """An object consisting of a line represented by a list of vertices
-    (3D positions). Some materials will render the line as a continuous line,
-    while other materials will consider each pair of points a segment.
+    """A line.
+
+    An object representing a line using a list of vertices (3D positions). Some
+    materials will render the line as a continuous line, while other materials
+    will consider each pair of consequtive points a segment.
+
+    The picking info of a Line (the result of ``renderer.get_pick_info()``) will
+    for most materials include ``vertex_index`` (int) and ``segment_coord``
+    (float, sub-segment coordinate).
+
+    Parameters
+    ----------
+    geometry : Geometry
+        The data defining the shape of the object.
+    material : Material
+        The data defining the appearence of the object.
+    visible : bool
+        Whether the object is visible.
+    render_order : int
+        The render order (when applicable for the renderer's blend mode).
+    render_mask : str
+        Determines the render passes that the object is rendered in. It's
+        recommended to let the renderer decide, using "auto".
+    position : Vector
+        The position of the object in the world. Default (0, 0, 0).
 
-    The picking info of a Line (the result of
-    ``renderer.get_pick_info()``) will for most materials include
-    ``vertex_index`` (int) and ``segment_coord`` (float, sub-segment coordinate).
     """
 
 
 class Points(WorldObject):
-    """An object consisting of points represented by vertices (3D positions).
+    """A point cloud.
+
+    An object consisting of points represented by vertices (3D positions).
```

Comment:
I think this is true for any object that is rendered. Ultimately, it is just a set of vertices with some extra data defined on them x)

Reply:
```suggestion
    """An object representing a disconnected cluster of individual points.

```

Maybe like this?

Reply:
> Is there a guideline on how short/long the "title" should be?

Yes there is. They should be a one-liner and not use the function name ([ref](https://numpydoc.readthedocs.io/en/latest/format.html#short-summary)). The main thing is to have something snappy that looks good when looking through the table in the docs. Otherwise we can do whatever we think is meaningful.

> I think some are so short they barely add anything over the class name.

Yes. sometimes its hard to say something useful beyond a descriptive class name. (guess it means you chose good variable names :D). It's just odd to not have a short-description because then there is a table entry that is randomly empty, which looks like we forgot something.

Reply:
> Maybe like this?

I'd say the original was more understandable for a broader audience.

Reply:
@Korijn The original as in the version that is currently in the PR, or the version that Almar suggested?

Reply:
> the version that is currently in the PR

:)

Reply:
👍